app/page-partials/admin-dashboard/MetadataEditModal.tsx

```diff
@@ -36,7 +36,7 @@ function MetadataEditModal({ request, onUpdate }: Props) {
   useEffect(() => {
     setUuid(request.idirUserid);
     setStatus(request.status);
-  }, [request]);
+  }, [request.id]);
```

Comment:
on the list page, there is a interval to refresh the list every certain time that force updating the the request in editing, to avoid the unexpected result, needed to just watchid field not whole request data.

as status or idirUserid are updating inside the modal, no need to watch those fields to update the modal.